src/distributions/bernoulli.rs

```diff
@@ -56,7 +56,7 @@ pub struct Bernoulli {
 // the RNG, and pay the performance price for all uses that *are* reasonable.
 // Luckily, if `new()` and `sample` are close, the compiler can optimize out the
 // extra check.
-const ALWAYS_TRUE: u64 = ::core::u64::MAX;
+const ALWAYS_TRUE: u64 = crate::core::u64::MAX;
```

Comment:
This is one of those little inconsistencies of the new module system... we have to use `crate::core` because it's locally renamed. Hopefully renaming imports will get better support in the future though.

In fact, semantically this isn't even a rename; if we could just have some `cfg` to enable `core` available with `std` then we could replace `crate::core` with just `core`.

Reply:
> This is one of those little inconsistencies of the new module system... we have to use crate::core because it's locally renamed.

Technically, we are doing the same as before. We can't just import directly from the `core` crate, because it does not exist outside of `no_std`.

IIRC, there is some discussion to replace `crate::` with just `::`, which would reduce verbosity and churn drastically, arguably at the cost of clarity.

Reply:
In the new edition you'll just be able to write `core::u64::MAX` without any imports because `core` is always available.